cli/src/pcluster/validators/scheduler_plugin_validators.py

```diff
@@ -29,6 +29,18 @@ def _validate(self, grant_sudo_privileges: bool, requires_sudo_privileges: bool)
             )
 
 
+class GrantSudoPrivilegesValidator(Validator):
+    """Verify SystemUsers does not have SudoerConfiguration set when GrantSudoPrivileges is true."""
+
+    def _validate(self, grant_sudo_privileges: bool, sudoer_configuration):
+        if not grant_sudo_privileges and sudoer_configuration:
+            self._add_failure(
+                "SudoerConfiguration of SystemUsers can not be set because GrantSudoPrivileges is not set or set to "
+                "false.",
```

Comment:
reword to: `The used scheduler plugin requires the creation of SystemUsers with sudo access. To grant such rights please set GrantSudoPrivileges to true.`

Reply:
Changed